test/parallel/test-sync-io-option.js

```diff
@@ -20,7 +20,7 @@ if (process.argv[2] === 'child') {
     execFile(process.execPath, args, function(err, stdout, stderr) {
       assert.equal(err, null);
       assert.equal(stdout, '');
-      if (/WARNING[\s\S]*fs\.readFileSync/.test(stderr))
+      if (/WARNING[\s\S]*readFileSync/.test(stderr))
```

Comment:
Maybe it's possible to modify `Environment::PrintSyncTrace` to get it back ?
It's not affecting the stack traces generated by V8 when an error is thrown.